.ci/dockerfiles/Dockerfile.xpu

```
@@ -0,0 +1,134 @@
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Set environment variables early
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/"
+ENV VLLM_TARGET_DEVICE=xpu
+ENV VLLM_WORKER_MULTIPROC_METHOD=spawn
+
+# suppress the python externally managed environment error
+RUN python3 -m pip config set global.break-system-packages true
+
+# Install UCX and dependencies
+RUN git clone ${UCX_REPO_URL} -b ${UCX_BRANCH} ${UCX_SRC_DIR} && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)